core/authentication_api.php

```diff
@@ -209,7 +209,7 @@ function auth_auto_create_user( $p_username, $p_password ) {
 
 	if( $t_auto_create ) {
 		# attempt to create the user
-		$t_cookie_string = user_create( $p_username, md5( $p_password ) );
+		$t_cookie_string = user_create( $p_username, "" );
```

Comment:
Set the password to:
crypto_generate_uri_safe_nonce( 32 ) instead of blank string.